src/Command/CommandBase.php

```diff
@@ -1396,7 +1397,7 @@ protected function promptChooseAcsfSite($cloud_environment) {
     foreach ($acsf_sites['sites'] as $domain => $acsf_site) {
       $choices[] = "{$acsf_site['name']} ($domain)";
     }
-    $choice = $this->io->choice('Choose a site', $choices);
+    $choice = $this->io->choice('Choose a site', $choices, $choices[0]);
```

Comment:
I might check to see if there's only one choice and just skip asking if that's the case.

Reply:
I thought about that but figured we'd leave it for later, because it's not trivial to get right. We'd want to make it really clear that only a single choice was available, and we wouldn't want to do it for longer or dangerous operation (i.e. `ide:delete`).